.github/workflows/pr_title_checker_config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "CHECKS": {
+    "regexp": "^(Revert \")?WT-[0-9]+ ",
```

Comment:
Do we ever commit something that is not `WT`? Looks too strict.

Reply:
Oh actually there is one exception here. Sometimes we'll do a bulk backports that merges multiple tickets into one. I'll see if I can find an example

Reply:
SGTM 👍